zombies.js

```diff
@@ -178,7 +267,21 @@
  * @name eat
  * @param {Food} itemToEat  The food item to eat.
  */
-
+eat(itemsToEat) {
+  let index = this._pack.indexOf(itemsToEat);
+  
+  if(itemsToEat instanceof Food && index !== -1) {
+    let newEnergy = this.health + itemsToEat.foodEnergy;
+    console.log(newEnergy);
+    this._pack.splice(index, 1);
+    
+    if(newEnergy < this.getMaxHealth()) {
```

Comment:
You could simplify this if conditional by just adding the food's energy to the player's health and resetting it to maxHealth if it is exceeded.